meson_options.txt

```diff
@@ -54,6 +54,13 @@ option(
   description: 'Whether to build support for KDE System Settings'
 )
 
+option(
+  'config-xdp',
+  type: 'boolean',
+  value: true,
```

Comment:
The relevance of this config module is for apps running 'as flatpaks' right? So a 'baremetal' install should not have this config module (but it has to be available to be added into the flatpak)?

Reply:
Yes, it must be part of the runtime. But there is no way to detect it, and i don't think maintainers are that smart :P As it causes no harm on non flatpak system, i would like to keep it.

Reply:
Hei, that was too easy... i'll switch it :)